Dockerfile

```diff
@@ -67,22 +51,25 @@ RUN yum update -y && \
         xrootd-python && \
     yum clean -y all
 
+# Configuration for CERN Open Data Portal instance:
+ENV APP_INSTANCE_PATH=/usr/local/var/cernopendata/var/cernopendata-instance
+ARG UWSGI_WSGI_MODULE=cernopendata.wsgi:application
```

Comment:
Can L56-63 be moved to happen after L74 (or even after L78) so in the case they change the Dockerfile might not need to be rebuild.

Reply:
not all, `APP_INSTANCE_PATH` needs to remain before `create-instance.sh` but is true that this we wont change, whereas tuning uWSGI parameters is likely to happen